components/brave_wallet_ui/components/desktop/popup-modals/sell-asset-modal/sell-asset-modal.style.ts

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+
+// Assets
+import WarningCircleFilled from '../../../../assets/svg-icons/warning-circle-filled.svg'
+
+// Shared Styles
+import { AssetIconProps, AssetIconFactory } from '../../../shared/style'
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 0 24px 24px 24px;
+`
+
+export const InputSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 27px 24px 24px 24px;
+  border-radius: 12px;
+  background-color: ${(p) => p.theme.color.background01};
+  margin-bottom: 16px;
+`
+
+export const Row = styled.div <{
```

Comment:
Can we reuse https://github.com/brave/brave-core/blob/71198d2d075cebf7498875e31c9145155373d271/components/brave_wallet_ui/components/shared/style.tsx#L101

Reply:
Reused!